FifthOrder:
  name: my_filter
  type: MultiChannelMeanFilterDouble
  params: {number_of_observations: 5}
